bodhi/server/models.py

```diff
@@ -1259,6 +1272,41 @@ def signed(self):
             return True
         return all([build.signed for build in self.builds])
 
+    @property
+    def ci_status(self):
+        """ Returns a boolean representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns False if at least one of the tests of the builds associated
+        with this update are queued, running or failed.
+        """
+
+        ci_status = set([build.ci_status for build in self.builds])
+        if not ci_status.difference(set([CiStatus.ignored, CiStatus.passed])):
+            return True
+        return False
```

Comment:
If you go with my suggestion on the other method to rename it and have it return an enum, you can also simplify this method to use the other one. Something like this maybe:

```
def ci_passed(self):
    if self.ci_status in (CiStatus.ignored, CiStatus.passed):
        return True
    return False
```

What do you think?